optionals/compat_sog/config.cpp

```diff
@@ -26,12 +26,19 @@ class CfgPatches {
             QGVAR(dshkm_AA_carry),
             QGVAR(rpd_carry),
             QGVAR(pk_carry),
+            QGVAR(mg42_carry),
+            QGVAR(sgm_carry),
+            QGVAR(sgm_shield_carry),
+            QGVAR(mk18_carry),
             QGVAR(mortar_m29_carry),
             QGVAR(mortar_m2_carry),
             QGVAR(mortar_type53_carry)
         };
         requiredVersion = REQUIRED_VERSION;
-        requiredAddons[] = {"vn_weapons", "vn_emm_main_menu", "weapons_f_vietnam_c", "ace_trenches"};
+        requiredAddons[] = {
+            "ace_trenches",
+            "loadorder_f_vietnam"
```

Comment:
`Warning Message: Addon 'ace_compat_sog' requires addon 'loadorder_f_vietnam'`
I get this when testing with `!Workshop\@S.O.G. Prairie Fire - Creator DLC Compatibility Data for Non-Owners`

it does have vn_weapons but not loadorder_f_vietnam, so I think we should keep the the old way

Reply:
The loadorder will be there soon and I'm holding up the PR as draft until it's there ;)